common/app/views/support/cleaner/AmpEmbedCleaner.scala

```diff
@@ -113,12 +131,20 @@ case class AmpEmbedCleaner(article: Article) extends HtmlCleaner {
     }
   }
 
+  private def createAmpSoundcloudElement(document: Document, trackId: String): Element = {
```

Comment:
this method should be within the cleanAmpSoundcloud one as it's not related to any of the others.  You could wrap them both in an object for readability.